src/jukebox/components/playermpd/__init__.py

```diff
@@ -340,6 +340,20 @@ def shuffle(self, random):
         # As long as we don't work with waiting lists (aka playlist), this implementation is ok!
         self.mpd_retry_with_mutex(self.mpd_client.random, 1 if random else 0)
 
+    @plugs.tag
+    def set_shuffle(self, option='toggle'):
```

Comment:
Should `shuffle` still be exported?
Or better remove `@plugs.tag` (and make internal `_shuffle`) and only use this method from external?